news/2020-04-20-monthly-beatmapping-contest-3-results.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Monthly Beatmapping Contest #3: Results"
+date: 2020-04-20 18:00:00 +0000
+---
+
+Check out the winning submissions from the third monthly beatmapping contest!
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+The **Monthly Beatmapping Contest** (***MBC***) has returned and been running strong since the start of the year! Take a peek at the [MBC wiki article](/wiki/Contests/Monthly_Beatmapping_Contest) for information on anything and everything MBC-related.
```

Comment:
"has returned and been running strong since" sounds off. sticking to one of those directions would probably sound better

something like: "The **Monthly Beatmapping Contest** (***MBC***) has returned yet again for another round of exciting entries!"